activestorage/app/controllers/concerns/active_storage/file_server.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ActiveStorage::FileServer # :nodoc:
+  extend ActiveSupport::Concern
```

Comment:
This module doesn’t use the features of `ActiveSupport::Concern` and doesn’t need to extend it.